lib/agent/agent-glue.c

```diff
@@ -536,6 +536,9 @@ frida_get_address_of_thread_create_func (void)
   return GUM_FUNCPTR_TO_POINTER (_beginthreadex);
 #elif defined (HAVE_DARWIN) || defined (HAVE_ANDROID)
   return GUM_FUNCPTR_TO_POINTER (pthread_create);
+#elif defined (HAVE_UCLIBC)
+  void * handle = dlopen ("libpthread.so.0", RTLD_GLOBAL | RTLD_LAZY);
```

Comment:
When should we dlclose this library?

> On 28 Jun 2016, at 02:28, Ole André Vadla Ravnås <notifications@github.com> wrote:
> 
> In lib/agent/agent-glue.c:
> 
> > @@ -536,6 +536,9 @@ frida_get_address_of_thread_create_func (void)
> >    return GUM_FUNCPTR_TO_POINTER (_beginthreadex);
> >  #elif defined (HAVE_DARWIN) || defined (HAVE_ANDROID)
> >    return GUM_FUNCPTR_TO_POINTER (pthread_create);
> > +#elif defined (HAVE_UCLIBC)
> > +  void * handle = dlopen ("libpthread.so.0", RTLD_GLOBAL | RTLD_LAZY);
> We also increase the reference-count on the library here – presumably we also want to dlclose() to avoid this?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>